src/desktop.c

```diff
@@ -292,6 +293,22 @@ desktop_focus_topmost_mapped_view(struct server *server)
 	desktop_move_to_front(view);
 }
 
+static struct wlr_surface *
+get_surface_from_layer_node(struct wlr_scene_node *node)
+{
```

Comment:
Do we want to add an `assert(node->data)` here?

Reply:
Missed this one. Have just added and re-force-pushed.